src/linux.rs

```diff
@@ -136,6 +137,22 @@ impl Command {
         self
     }
 
+    /// Reassociate child process with a namespace specified by a file
+    /// descriptor
+    ///
+    /// `fd` argument is a file descriptor referring to a namespace
+    ///
+    /// 'ns' is an optional namespace type. `None` allows any type of
+    /// namespace
+    ///
+    /// See `man 2 setns` for further details
+    pub fn setns(&mut self, fd: RawFd, ns: Option<Namespace>)
+        -> &mut Command
+    {
+        self.config.setns_namespaces.push((fd, ns));
```

Comment:
Well, so this looks more obvious, right?

At the end of the day, we don't need to support 100% of possible calls of `setns` just 99% of the real use cases. For now, I think dropping support for `None` is viable.